src/dataServices/dynamoDbDataService.ts

```diff
@@ -217,9 +201,10 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
     }
 
     async initiateExport(initiateExportRequest: InitiateExportRequest): Promise<string> {
+        this.assertValidTenancyMode(initiateExportRequest.tenantId);
         await this.throttleExportRequestsIfNeeded(initiateExportRequest.requesterUserId);

```

Comment:
The `throttleExportRequestsIfNeeded` needs to account for multi-tenancy. I missed this on the design doc. 

The basic idea is that each tenant gets their own allowed `MAXIMUM_SYSTEM_LEVEL_CONCURRENT_REQUESTS` instead of all of them sharing the same requests count. Otherwise "noisy neighbors" can prevent small tenants from starting export requests.

I *think* that this is not a big change

Reply:
Yes, it makes sense and it's not a big change. I'll make the update.